pkg/local_workflows/code_workflow/code_client_helper.go

```diff
@@ -25,3 +28,18 @@ func (c *codeClientConfig) SnykCodeApi() string {
 func (c *codeClientConfig) SnykApi() string {
 	return c.localConfiguration.GetString(configuration.API_URL)
 }
+
+func (c *codeClientConfig) SnykCodeAnalysisTimeout() time.Duration {
```

Comment:
Question: the newly added function `SnykCodeAnalysisTimeout()` is not used somewhere, is this expected?

Reply:
What do you mean it's not used? This is where it's used in `code-client-go` https://github.com/snyk/code-client-go/blob/main/internal/analysis/analysis.go#L349

Reply:
Should it be used in the native workflow?

Reply:
`SnykCodeAnalysisTimeout()` should be used for the new code workflow in GAF that uses ignores, which from what I can tell from the code is the native workflow. Did I misunderstand anything?

Reply:
Sorry I missed that this change is in the injected configuration object 🤦 now the world makes sense again.